src/devices/src/virtio/block/device.rs

```diff
@@ -146,6 +197,7 @@ pub struct Block {
     pub(crate) partuuid: Option<String>,
     pub(crate) root_device: bool,
     pub(crate) rate_limiter: RateLimiter,
+    pub(crate) cache_type: CacheType,
```

Comment:
this is already part of `DiskProperties`, why duplicate here?

Reply:
You're right, I removed the duplicate.